src/tray/vars/all/main.yml

```diff
@@ -41,19 +41,29 @@ marsha_cloudfront_private_key_path: "/private/.ssh/aws/ssh-privatekey"
 # signature with the aforementioned SSH key
 marsha_should_sign_requests: true
 marsha_activate_http_basic_auth: false
-marsha_deploy_clean_medialive_dev_stack_cronjob: false
-marsha_check_harvested_cronjob_schedule: "0 3 * * *"
-marsha_check_live_state_cronjob_schedule: "*/20 * * * *"
-marsha_check_idle_state_cronjob_schedule: "0 4 * * *"
-marsha_clean_mediapackages_cronjob_schedule: "0 5 * * *"
-marsha_clean_medialive_dev_stack_cronjob_schedule: "*/5 * * * *"
-marsha_clean_aws_elemental_stack_cronjob_schedule: "0 */2 * * *"
-marsha_prefetch_saml_fer_metadata_cronjob_schedule: "0 */6 * * *"
+
+# -- Cronjob pipeline
+
+# List of cronjob to add in marsha
+# marsha_cronjobs:
+#   - name: clean-aws-elemental-stack
+#     enabled: "{% if env_type == 'production' %}false{% else %}true{% endif %}"
+#     schedule: "0 */2 * * *"
+#     command:
+#       - "/bin/bash"
+#       - "-c"
+#       - python manage.py clean_aws_elemental_stack
+#     resources:
+#       requests:
+#         cpu: 50m
+#         memory: 500Mi
+marsha_cronjobs: []
```

Comment:
Shouldn't we already fill marsha_cronjobs with deleted cronjob values ?

Reply:
The idea is to not maintain a list here but to do it in matsuo for each environment. Doing this we don't have to wait for a new release to deploy a new cronjob